### PR TITLE
Bug fixes for Ursula: availability sensor, cert/stamp verification, nicknames

### PR DESCRIPTION
**Pull request by KPrasch**
Fixes several regressions introduced in #1462 and #1697 

Advances fixes for https://github.com/nucypher/nucypher/issues/1698, #1823, and #1820 

Tested in live production environment.

### CODE REVIEW

**Comment by derekpierre** [comment]
@KPrasch a reminder of my comment in #1462: https://github.com/nucypher/nucypher/pull/1462#discussion_r401049059

Basically, if each failed check results in the reduction of the score by a factor of 0.9:
`new_score = 0 + 0.9 * <previous_score>`
then we would never get a score of 0 for the `shutdown_threshold`. Instead, we could set the `shutdown_threshold` to some low non-zero value. For example, setting the shutdown threshold to be `1` would require 22 consecutive failed checks (assuming ursulas start at a score of `10`) for the score to become <= 1 and force a shutdown.

File: nucypher/network/sensors.py [hunk 1]
```diff
@@ -43,31 +43,36 @@ def __init__(self, ursula, enforce_loneliness: bool = True):
             9: self.mild_warning,
             7: self.medium_warning,
             2: self.severe_warning,
-            0: self.shutdown_everything
+            1: self.shutdown_everything  # 0 is unobtainable
```
Comment (KPrasch) [inline nucypher/network/sensors.py:46]: @derekpierre :heavy_check_mark:

File: nucypher/cli/painting.py [hunk 2]
```diff
@@ -517,7 +521,12 @@ def paint_stakes(emitter, stakeholder, paint_inactive: bool = False, staker_addr
         missing = staker.missing_confirmations
         min_reward_rate = prettify_eth_amount(staker.min_reward_rate)
 
-        staker_data = [f'Missing {missing} confirmation{"s" if missing > 1 else ""}' if missing else f'Confirmed #{last_confirmed}',
+        if missing == -1:
+            missing_info = "Never Confirmed (New Stake)"
+        else:
+            missing_info = f'Missing {missing} confirmation{"s" if missing > 1 else ""}' if missing else f'Confirmed #{last_confirmed}'
```
Comment (cygnusv) [inline nucypher/cli/painting.py:527]: I think we can unroll these conditionals too.

**Comment by derekpierre** [comment]
Some subsequent comments from @tuxxy regarding the scale of the scores.

```
I just mean that beginning at 100 makes the number more natural/fine.
...
At 10, with 7 missed checks, you're < 5. At 100, with 7 missed checks, you're < 50.
...
the distance between a score of 3 to 5 feels much smaller than it actually is.
compared to a score of 30 to 50.
or even 300 to 500.
```

**Pull request merged**